support-frontend/assets/components/thankYou/signIn/signInItems.tsx

```diff
@@ -123,10 +123,9 @@ export function SignInBodyCopy(): JSX.Element {
 export function SignInCTA({ email, csrf }: SignInBodyCopyProps): JSX.Element {
 	const [signInUrl, setSignInUrl] = useState('https://theguardian.com');
 
-	useEffect(() => {
-		const payload = {
-			email,
-		};
+	function fetchSignInLink(payload: { email: string }) {
+		if (window.location.hostname !== 'theguardian.com') return;
```

Comment:
What's the reason for this new window.location.hostname conditional?

Reply:
Same as above, we were seeing errors in the storybook console and thought we wouldn't want to do this is the dev env